### PR TITLE
Added Homepage , Category and Quiz Pages

### PR DESCRIPTION
**Pull request by ayush65**
Added Homepage  , Added Category , Added Quiz Pages

### CODE REVIEW

File: src/Quizlisting/Quizlisting.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+  const [timer, setTimer] = useState(45);
+
+  const intervalFunction = (times) => {
+    let j = 0;
+    for (let i = times; i >= 0; i--) {
+      j = j + 1;
+      setTimeout(() => {
+        console.log(i);
```
Comment (narharikale) [inline src/Quizlisting/Quizlisting.js:16]: console.log here

File: src/Quizlisting/Quizlistingfriends2.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom';
+import Navbar from '../Navbar/Navbar';
+
+function Quizlistingfriends2() {
+    const [timer, setTimer] = useState(45);
+
+    const intervalFunction = (times) => {
```
Comment (narharikale) [inline src/Quizlisting/Quizlistingfriends2.js:8]: you using intervalFunction many time , so you can make utils for that

File: src/Quizlisting/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import Quizlisting from './Quizlisting';
```
Comment (narharikale) [inline src/Quizlisting/index.js:1]: can we make only one component for all these quiz listing

**Review by narharikale — APPROVED** [review]
good work added some suggestions

**Review by anjali1102 — APPROVED** [review]

File: src/Quizlisting/index.js (hunk 3 above)
Comment (ayush65) [reply src/Quizlisting/index.js]: yea this pr was for the static pages for the quiz app

Comment (ayush65) [reply src/Quizlisting/index.js]: will implement that in  next pr

File: src/Quizlisting/Quizlisting.js (hunk 1 above)
Comment (ayush65) [reply src/Quizlisting/Quizlisting.js]: updated

**Pull request merged**